### ISSUE TITLE
Suppressing library messages and other output from nodes

### ISSUE THREAD

**Issue by mw55309** [hadley/plyr#269]
Hi

I'm using doParallel to create the cluster, directing output to a file:

workers=makeCluster(2, outfile="parallel.fastq")

Then running a_ply to apply a function to a bunch of files:

a_ply(array(f5files), 1, extractfastq, .print=TRUE, .parallel=TRUE, .paropts=list(.packages=c("poRe","bit64","rhdf5")))

This actually works fine, and gives the output I want, except at the top of parallel.fastq is all of the normal R output that happens when libraries are loaded, eg:

Loading required package: plyr
loaded plyr and set parent environment
Loading required package: rhdf5
Loading required package: plyr
loaded plyr and set parent environmen

 as well as a couple of other messages:

starting worker pid=15480 on localhost:11214 at 21:08:07.376
starting worker pid=15489 on localhost:11214 at 21:08:07.745

Basically what I want is for all of this output to be suppressed, but the output from my function to remain.

Is this possible?

Cheers
Mick
